packages/ctl/ctl.js

```diff
@@ -104,6 +106,99 @@ masterConfigCommands.add(new libCommand.Command({
 		await libLink.messages.setMasterConfigProp.send(control, request);
 	},
 }));
+masterConfigCommands.add(new libCommand.Command({
+	definition: ["edit [editor]", "Edit master configuration", (yargs) => {
+		yargs.positional("editor", { describe: "Editor to use",
+			type: "string",
+			default: "" });
+	}],
+	handler: async function(args, control) {
+		let response = await libLink.messages.getMasterConfig.send(control);
+		let tmpFile = await libFileOps.getTempFile("ctl-", "-tmp", os.tmpdir());
+		let editor = 0;
+		if (args.editor) {
+			editor = args.editor;
+			/* eslint-disable */
+		} else if (process.env.EDITOR) {
+			editor = process.env.EDITOR;
+		} else if (process.env.VISUAL) {
+			editor = process.env.VISUAL
+			/* eslint-enable */
+			// needed for the process.env statements to not be flagged by eslint
+			// prio for editors is CLI arg > env.EDITOR > env.VISUAL
+		} else {
+			throw new libErrors.CommandError("No editor avalible. Checked CLI input, EDITOR and VISUAL env vars");
```

Comment:
Maybe show example of how it should have been used, ex `use --editor nano` or something?